<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="generator" content="pandoc" />
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <meta name="author" content="Nathan Brouwer" />
    <meta name="date" content="2023-01-05" />
    <title>Nathan L. Brouwer, Phd</title>

    
        <link href="resume_files/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
        <link href="resume_files/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />
        <link href="resume_files/paged-0.19/css/resume.css" rel="stylesheet" />
        <script src="resume_files/paged-0.19/js/config.js"></script>
        <script src="resume_files/paged-0.19/js/paged.js"></script>
        <script src="resume_files/paged-0.19/js/hooks.js"></script>
    
    
  </head>
  <body>

<div id="aside" class="section level1">
<h1>Aside</h1>
<div id="contact" class="section level2">
<h2>Contact Info</h2>
<ul>
<li><i class="fa fa-envelope"></i> <a href="mailto:brouwern@gmail.com" class="email">brouwern@gmail.com</a></li>
<li><i class="fa fa-github"></i> <a href="https://github.com/brouwern">github.com/brouwern</a></li>
<li><i class="fa fa-phone"></i> 517-898-5440</li>
</ul>
</div>
<div id="skills" class="section level2">
<h2>Relevant Skills</h2>
<ul>
<li><p><strong>STATISTICAL MODELING</strong>: Multiple publications employing <strong>GLMMs</strong> and <strong>GAMMs</strong>.</p></li>
<li><p><strong>DATA VIZUALIZATION</strong>: Expert exploring &amp; visualizing complex data with <strong><code>ggplot2</code></strong>.</p></li>
<li><p><strong>DATA CLEANING</strong>: Expert in <strong>data manipulation</strong> &amp; cleaning, including use of <strong>Tidyverse</strong>.</p></li>
<li><p><strong>UNSUPERVISED MACHINE LEARNING</strong>: Skilled with identifying patterns in <strong>high-dimensional data</strong> with ordination &amp; cluster analysis.</p></li>
<li><p><strong>SCIENTIFIC COMMUNICATION</strong>: Excellent written, verbal &amp; graphical communication skills.</p></li>
<li><p><strong>R PACKAGES</strong>: Experience using R packages to distribute scientific software and organize complex analyses.</p></li>
<li><p><strong>REPRODUCIBLE WORKFLOWS</strong>: Large portfolio of <strong>RMarkdown</strong> products, including books &amp; package documentation.</p></li>
</ul>
</div>
</div>
<div id="main" class="section level1">
<h1>Main</h1>
<div id="title" class="section level2">
<h2>Nathan L. Brouwer Phd</h2>
<div id="computational-ecologist-specializing-in-complex-datasets" class="section level3">
<h3>Computational ecologist specializing in complex datasets</h3>
<p>I am a computational ecologist, data scientist and educator with 15 years of experience extracting insights from messy data and communicating the results. I specialize in using generalized-linear mixed models (GLMMs) to analyze data from long-term observational studies. Since my PhD I have also taught myself key aspects of bioinformatics, phylogenetics, machine learning and population genomics.</p>
</div>
</div>
<div id="education" class="section level2" data-icon="graduation-cap" data-concise="true">
<h2>Education</h2>
<div id="seattle-pacific-university" class="section level3">
<h3>Seattle Pacific University</h3>
<p>B.S. in Biology</p>
<p>N/A</p>
<p>2002</p>
</div>
<div id="university-of-pittsburgh" class="section level3">
<h3>University of Pittsburgh</h3>
<p>Phd in Biological Sciences</p>
<p>N/A</p>
<p>2015</p>
<p>Dissertation: <a href="http://d-scholarship.pitt.edu/26457/">Applying multilevel longitudinal models to plant demographic processes:
novel insights into the long-term impacts of invasive species and overabundant herbivores.</a></p>
</div>
</div>
<div id="research-professional-experience" class="section level2" data-icon="laptop">
<h2>Research &amp; Professional Experience</h2>
<div id="lecturer---general-quantitative-biology" class="section level3">
<h3>Lecturer - General &amp; Quantitative Biology</h3>
<p>Unv. of Pittsburgh Dept. of Biological Sciences</p>
<p>N/A</p>
<p>2019-present</p>
<ul>
<li>Develop and deliver curriculum for updated <strong>Computational Biology</strong> course</li>
<li>Instructor for <strong>biostatistics</strong>, writing, intro biology courses &amp; intro biology labs.</li>
<li>Consult on <strong>statistics curriculum</strong> development for laboratory classes.</li>
</ul>
</div>
<div id="post-doctoral-research-associate" class="section level3">
<h3>Post-doctoral Research Associate</h3>
<p>National Aviary</p>
<p>Pittsburgh, PA</p>
<p>2015 - 2019</p>
<ul>
<li>Use GLMMs to analyze decade-long tropical bird population &amp; community datasets.</li>
<li>Convert published models on migratory birds into an R package &amp; develop methods</li>
<li>Develop methods for sensitivity analysis and uncertainty propagation for migration models</li>
</ul>
</div>
<div id="adjunct-professor---biology" class="section level3">
<h3>Adjunct Professor - Biology</h3>
<p>La Roche College</p>
<p>Pittsburgh, PA</p>
<p>2018-2019 (Fall &amp; Winter)</p>
<ul>
<li>Co-teach introduction to research course (fall)</li>
<li>Develop new lab and data analysis activities</li>
<li>Teach writing for biological sciences (winter)</li>
</ul>
</div>
<div id="adjunct-professor---biological-data-analysis" class="section level3">
<h3>Adjunct Professor - Biological Data Analysis</h3>
<p>Dusquesne Unv. &amp; California Unv. of PA</p>
<p>N/A</p>
<p>2017-2018</p>
<ul>
<li>Dusquesne: Teach graduate biostatistics lecture course &amp; consult with students on data analysis (Spring 2018)</li>
<li>CalU: Teach undergraduate biostatistics lecture course &amp; develop curriculum for new R-based lab</li>
</ul>
</div>
<div id="graduate-research-assistant" class="section level3">
<h3>Graduate Research Assistant</h3>
<p>Department of Biological Sciences, University of Pittsburgh</p>
<p>N/A</p>
<p>2010 - 2015</p>
<ul>
<li>Manage and analyze decade-long plant population data.</li>
<li>Design &amp; conduct field research on plant demography.</li>
</ul>
</div>
<div id="peace-corps-agroforestry-volunteer" class="section level3">
<h3>Peace Corps Agroforestry Volunteer</h3>
<p>National Agricultural Research Institute</p>
<p>The Gambia, West Africa</p>
<p>2004 - 2006</p>
<ul>
<li>Assist in staff development, including data analysis and experimental design</li>
</ul>
</div>
<div id="infecious-disease-research-scientist" class="section level3">
<h3>Infecious Disease Research Scientist</h3>
<p>University of Washington Department of Allergy &amp; Infectious Disease</p>
<p>N/A</p>
<p>2002 - 2004</p>
</div>
</div>
<div id="science-communication" class="section level2" data-icon="file">
<h2>Science Communication</h2>
<div id="computational-biology-for-all-an-open-access-book-for-bioinformatics-computational-biology-vs-0.9" class="section level3">
<h3><a href="https://brouwern.github.io/compbio4all_book/">Computational Biology for All! An open access book for bioinformatics &amp; computational biology vs 0.9</a></h3>
<p>Open-access computational biology textbook</p>
<p>N/A</p>
<p>2022</p>
<p><strong>Brouwer</strong></p>
</div>
<div id="a-little-book-of-r-for-bioinformatics-vs.-2.0" class="section level3">
<h3><a href="https://brouwern.github.io/lbrb/">A Little Book of R for Bioinformatics vs. 2.0</a></h3>
<p>Open-access bioinformatics primer.</p>
<p>N/A</p>
<p>2022</p>
<p>Coghlan (au.) &amp; <strong>Brouwer</strong> (ed., au)</p>
</div>
</div>
<div id="r-packages" class="section level2" data-icon="file">
<h2>R Packages</h2>
<div id="population-modeling-redstart-an-r-package-for-periodic-full-annual-cycle-avian-population-models-and-monte-carlo-simulation." class="section level3">
<h3>Population Modeling: <a href="https://github.com/brouwern/FACavian">redstart: An R package for Periodic Full-Annual Cycle Avian Population Models and Monte-Carlo simulation.</a></h3>
<p>R implementation and replication of Runge &amp; Marra (2005) Modeling Seasonal Interactions in the Population Dynamics of Migratory Birds.</p>
<p>N/A</p>
<p>2021</p>
<p><strong>Brouwer</strong> <em>et al.</em></p>
<ul>
<li>Website &amp; Tutorials: brouwern.github.io/FACavian/index.html</li>
</ul>
</div>
</div>
<div id="selected-publications" class="section level2" data-icon="file">
<h2>Selected Publications</h2>
<div id="population-models-direct-effects-of-a-non-native-invader-erode-native-plant-fitness-in-the-forest-understory" class="section level3">
<h3>Population Models: <a href="https://besjournals.onlinelibrary.wiley.com/doi/abs/10.1111/1365-2745.13233">Direct effects of a non-native invader erode native plant fitness in the forest understory</a></h3>
<p>Journal of Ecology 108:189–198 / <a href="https://datadryad.org/stash/dataset/doi:10.5061/dryad.4k7d5n7">Dryad</a></p>
<p>N/A</p>
<p>2019</p>
<p>Bialic-Murphy, <strong>Brouwer</strong> &amp; Kaliz.</p>
</div>
<div id="nmds-stream-acidification-reduced-availability-of-pollution-sensitive-aquatic-insects-are-associated-with-dietary-shifts-in-a-stream-dependent-neotropical-migratory-songbird." class="section level3">
<h3>NMDS: <a href="doi.org/10.7717/peerj.5141">Stream acidification &amp; reduced availability of pollution-sensitive aquatic insects are associated with dietary shifts in a stream-dependent Neotropical migratory songbird.</a></h3>
<p>PeerJ 6:e5141 / <a href="https://peerj.com/articles/5141/#supplemental-information">Data &amp; Code</a></p>
<p>N/A</p>
<p>2019</p>
<p>Trevelline, Nuttle, Porter, <strong>Brouwer</strong> <em>et al.</em></p>
</div>
<div id="nmds-dna-metabarcoding-reveals-the-importance-of-aquatic-prey-subsidies-the-structure-of-dietary-niches-in-a-community-of-breeding-riparian-songbirds." class="section level3">
<h3>NMDS: <a href="https://link.springer.com/article/10.1007/s00442-018-4136-0">DNA metabarcoding reveals the importance of aquatic prey subsidies &amp; the structure of dietary niches in a community of breeding riparian songbirds.</a></h3>
<p>Oecologia 187: 85-98.</p>
<p>N/A</p>
<p>2018</p>
<p>Trevelline, <strong>Brouwer</strong> <em>et al.</em></p>
</div>
<div id="glmm-avian-community-characteristics-demographics-reveal-how-conservation-value-of-regenerating-tropical-dry-forests-changes-with-forest-age" class="section level3">
<h3>GLMM: <a href="doi.org/10.7717/peerj.5217">Avian community characteristics &amp; demographics reveal how conservation value of regenerating tropical dry forests changes with forest age</a></h3>
<p>PeerJ 6: e5217 / Data and Code: <a href="https://github.com/brouwern/DRmencia">GitHub</a></p>
<p>N/A</p>
<p>2018</p>
<p>Latta, <strong>Brouwer</strong> <em>et al.</em></p>
</div>
<div id="glmm-long-term-monitoring-reveals-an-avian-species-credit-in-secondary-forest-patches-of-costa-rica" class="section level3">
<h3>GLMM: <a href="https://doi.org/10.7717/peerj.3539">Long-term monitoring reveals an avian species credit in secondary forest patches of Costa Rica</a></h3>
<p>PeerJ 6: e3539 / Data and Code: <a href="https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/CEVZPL">Harvard Dataverse</a></p>
<p>N/A</p>
<p>2017</p>
<p>Latta, <strong>Brouwer</strong> <em>et al.</em></p>
</div>
<div id="gamm-increased-photosynthetic-performance-of-an-invasive-forest-herb-mediated-by-deer-overabundance." class="section level3">
<h3>GAMM: <a href="https://academic.oup.com/aobpla/article/9/2/plx011/3074888">Increased photosynthetic performance of an invasive forest herb mediated by deer overabundance.</a></h3>
<p>AoB Plants 9: plx011 / <a href="https://academic.oup.com/aobpla/article/9/2/plx011/3074888#supplementary-data">Data &amp; Code</a></p>
<p>N/A</p>
<p>2017</p>
<p>Heberling, <strong>Brouwer</strong> &amp; Kalisz.</p>
</div>
<div id="glmm-mutualism-disrupting-allelopathic-invader-drives-carbon-stress-vital-rate-decline-in-a-forest-perennial-herb." class="section level3">
<h3>GLMM: <a href="10.1093/aobpla/plv014">Mutualism-disrupting allelopathic invader drives carbon stress &amp; vital rate decline in a forest perennial herb.</a></h3>
<p>AoB Plants 7: plv014</p>
<p>N/A</p>
<p>2015</p>
<p><strong>Brouwer</strong>, Hale &amp; Kalisz.</p>
</div>
</div>
</div>

<script>
(function() {
  var i, s, j, el, els;
  var create_el = function(type, className, content, isHTML) {
    var el = document.createElement(type);
    if (className) el.className = className;
    if (content) {
      if (isHTML) {el.innerHTML = content;} else {el.innerText = content;}
    }
    return el;
  }

  // replace h2 title with h1
  var title = document.getElementById('title').querySelector('h2');
  title.parentNode.replaceChild(create_el('h1', false, title.innerHTML, true), title);

  // add the class .aside to #aside
  el = document.getElementById('aside');
  if (el) el.className += ' aside';

  // tweak class names of sections, and add icons
  els = document.getElementById('main').querySelectorAll('.level2');
  for (i = 0; i < els.length; i++) {
    el = els[i];
    if (i === 0 && el.id === 'title') continue;
    el.className += ' main-block';
    if (el.dataset['concise']) el.className += ' concise';
    // if there is no icon, add an icon:
    if (el.firstElementChild.firstChild && el.firstElementChild.firstChild.nodeName !== 'I') {
      s = el.dataset['icon'] || 'bookmark';
      el.firstElementChild.insertBefore(create_el('i', 'fa fa-' + s), el.firstElementChild.firstChild);
    }
  }

  // tweak class names of blocks in sections
  els = document.getElementById('main').querySelectorAll('.level3');
  for (i = 0; i < els.length; i++) {
    el = els[i];
    if (el.parentNode.id === 'title') continue;
    el.className += ' blocks';
    el.innerHTML = '<div class="date"></div>' + '<div class="decorator"></div>' +
      '<div class="details"><header></header><div></div></div>' + el.innerHTML;
    var ps = el.querySelectorAll('p');
    // move the date paragraph to the date div
    if (ps.length >= 3) {
      s = ps[2].innerText;
      s = s === 'N/A' ? [] : s.split(' - ');
      el.removeChild(ps[2]);
      for (j = 0; j < s.length && j < 2; j++) {
        el.children[0].appendChild(create_el('span', false, s[j]))
      }
    }
    // move title, description, location to the details div
    (function(h) {
      h.appendChild(el.children[3]);
      var p = el.children[3];  // description
      if (p.innerText !== 'N/A') {
        h.appendChild(create_el('span', 'place', p.innerHTML, true));
      }
      el.removeChild(p);
      p = el.children[3];  // location
      if (p.innerText !== 'N/A') {
        s = create_el('span', 'location', p.innerHTML, true);
        s.innerHTML = '<i class="fa fa-map-marker-alt"></i> ' + s.innerHTML;
        h.appendChild(s);
      }
      el.removeChild(p);
    })(el.children[2].firstElementChild);
    // move the rest of content in a block into the last div of the details div
    s = el.children[2];
    while (j = s.nextSibling) {
      if (j.className === 'aside') { s = j; continue; }
      el.children[2].lastElementChild.append(j);
    }
  }
})();
</script>


  </body>
</html>
